### ISSUE TITLE
Home -> Header Incorrect Sub-Menu Container Type 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#759]
When the “Help Center” button has been activated to show the sub-menu, the content is contained inside a < div > element.  This means that there is no indication of the grouping of the sub-menu’s controls for screen reader users, nor can they skip to the beginning or end of the sub-menu directly.  It also means that the accessible label applied to the container is never communicated to users, because as a rule, screen readers will not read labels which are applied to < div > containers which don’t have an indicated role. 
 
Note: this observation also applies to the account controls sub-menu which can be expanded once a user has signed in.

Suggested Solution 
 
Exchange the < div > with a className attribute of “dropdown-menu dropdown-menu-lgright dropdown-secondary” for a < ul >. 
 
Side note: as per the advice in the “Confusing Button Labelling Pattern” section of this document, the off-screen span with the “Help Center” label should have been removed.  The value of the aria-labelledby attribute applied to the sub-menu references the ID of that < span >, and should be modified to reference the ID of the “Help Center” < button > instead. 
 
Proposed Code Sample 
`<ul className="dropdown-menu dropdown-menu-lg-right dropdown-secondary" arialabelledby={questionBtnId}>  … </ul>`

**Comment by rupeshparab** [comment]
this has been fixed while working on https://github.com/benetech/MathShare/issues/752

**Comment by jscholes** [comment]
@rupeshparab - in ProblemHeader, the "More Options" `<ul>` doesn't have an accessible label.  It needs an `aria-labelledby` attribute as with the "Help Center" dropdown in Home/Header.

```
<ul
	className="dropdown-menu dropdown-menu-lg-right dropdown-secondary"
	aria-labelledby={questionBtnId}
>
```

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/4e0daf9a92a28a91e46b69f09767b0928bd66315.
